src/main/kotlin/com/team4099/robot2023/subsystems/superstructure/SuperStructureRequest.kt

```diff
@@ -0,0 +1,17 @@
+package com.team4099.robot2023.subsystems.superstructure
+
+import org.team4099.lib.units.derived.Angle
+import org.team4099.lib.units.derived.ElectricalPotential
+
+// typealias GroundIntakeRequest = SuperStructureState.GroundIntakeStructure.GroundIntakeRequest
+// typealias GroundIntakeState = SuperStructureState.GroundIntakeStructure.GroundIntakeState
+// typealiasing for nested interfaces and sealed classes doesn't work
+// https://youtrack.jetbrains.com/issue/KT-34281/Access-nested-classes-including-sealed-class-subclasses-through-typealias
+
+sealed interface SuperStructureRequest {
+  // Implements SuperStructureRequest to ensure standardized structure
+  sealed interface GroundIntakeRequest : SuperStructureRequest {
```

Comment:
Why is this in `SuperstructureRequest`? I don't think you should be able to request a ground intake state from the superstructure. The lower level state machines should be fully abstracted underneath the superstructure (as in, the ground intake stuff should be fully independent of the superstructure stuff)

Reply:
agreed. this is just bad naming. a better name would be `RequestStructure` i thought it would be worth for subsystem requests to implement a general request structure in case we wanted all of them to share a function idk. doing this now just makes it easier to do it later and otherwise this has no effect but can remove